plugin.go

```diff
@@ -2,31 +2,32 @@
 package plasmactlbump
 
 import (
-	"github.com/spf13/cobra"
-
 	"github.com/launchrctl/keyring"
 	"github.com/launchrctl/launchr"
+	"github.com/spf13/cobra"
+
+	"github.com/skilld-labs/plasmactl-bump/pkg/sync"
```

Comment:
- [ ] WARNING  --skip-not-versioned option is deprecated, it will be used by default
      > let's remove this warning as we should already throw error during sync if resource is not versioned

Reply:
https://github.com/launchrctl/compose/pull/52
returned option here